content/docs/guides/crosswalk/kubernetes/introduction/00-prerequisites.md

```diff
@@ -1,15 +1,14 @@
 ---
 title: Prerequisites
 
-aliases: ["/docs/quickstart/k8s-the-prod-way/prerequisites/"]
+aliases: ["/docs/guides/crosswalk/kubernetes/prerequisites/"]
 ---
 
 This tutorial leverages [Pulumi][pulumi] and [Node.js][nodejs] to provision and configure
 infrastructure on AWS, Azure, or GCP. Pulumi is a platform for building and deploying cloud
 infrastructure and applications in your favorite language on any cloud. This particular tutorial is
 written in [TypeScript][ts], a strict superset of JavaScript that allows users to add optional type
-hints to variables. (Pulumi also supports Python and nothing in principle prevents using that
-instead.)
+hints to variables.
```

Comment:
Regarding the "Pulumi CLI" section below, it feels like this is duplicative with our existing installation instructions? That said, I was surprised to see that https://www.pulumi.com/docs/get-started/install/ doesn't list prerequisites like Yarn (though we should be equally as supportive of NPM). I know the getting started flow covers the language-specific prerequisites, but it's unfortunate we don't have one place for these. @justinvp is that intentional?

Reply:
I'm not tied to having this page, so I've updated the prereqs link point to https://www.pulumi.com/docs/get-started/kubernetes/install-pulumi/ instead of this page.